api/src/main/java/org/apache/iceberg/util/StructProjection.java

```diff
@@ -42,7 +42,7 @@
    */
   public static StructProjection create(Schema schema, Set<Integer> ids) {
     StructType structType = schema.asStruct();
-    return new StructProjection(structType, TypeUtil.select(structType, ids));
+    return new StructProjection(structType, TypeUtil.project(structType, ids));
```

Comment:
Looks like there aren't any uses of this call, which is good. I agree that we probably want this to use `project` instead of `select`.